samples/README.md

```diff
@@ -1,52 +1,41 @@
-[//]: # "This README.md file is auto-generated, all changes to this file will be lost."
-[//]: # "To regenerate it, use `python -m synthtool`."
 <img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google Cloud Platform logo" title="Google Cloud Platform" align="right" height="96" width="96"/>
 
-# [Google Cloud Internet of Things (IoT) Core: Node.js Samples](https://github.com/googleapis/nodejs-iot)
+# Google Cloud IoT Core NodeJS Samples
```

Comment:
This file gets generated by `synthtool`.  Any of the changes here would get overridden during our nightly autosynth run.  I don't know if this specific page supports mixin templates like the root README....  @bcoe ?

Reply:
To the theme of "Let it snow" ... "let it pave...." I'll let these be generated over the existing ones given I'm unclear of how to ensure it wouldn't happen.